sds_data_manager/lambda_code/SDSCode/pipeline_lambdas/batch_starter.py

```diff
@@ -65,6 +66,15 @@
     },
 ]
 
+SCHEDULED_JOBS = {
```

Comment:
Maybe extract this out into a `dependency_config.csv`-like file so we can manage it external to the code and it would be a list of these. Potentially even including the cron-string of what schedule you want to use.